.circleci/config.yml

```diff
@@ -2,21 +2,21 @@ version: 2.1
 jobs:
   build:
     machine:
-      image: 'ubuntu-1604:201903-01'
+      image: 'ubuntu-1604:201903-01' # 'ubuntu-2004:202010-01'
```

Comment:
Have we reverted back to the original ubuntu version?

Reply:
Yes. We reverted back to the Ubuntu16 container because of the time to build (plus one test failed on the container, but not cloudlab).

Reply:
I will open another issue for this.